client/state/selectors/get-jetpack-onboarding-settings.js

```diff
@@ -5,6 +5,14 @@
  */
 import { get } from 'lodash';
 
+/**
+ * Returns the Jetpack onboarding settings of a particular site.
+ * Returns null if site is not known.
```

Comment:
Extra nit: I'd use Returns null it the site is unknown. in place of Returns null if site is not known. and a given site instead of a particular site.

Reply:
Good suggestions, thanks 👍  54c1e3b6ff